opus_config.cmake

```diff
@@ -0,0 +1,73 @@
+include(opus_functions.cmake)
+
+get_library_version(OPUS_LIBRARY_VERSION OPUS_LIBRARY_VERSION_MAJOR)
+message(STATUS "Opus library version: ${OPUS_LIBRARY_VERSION}")
+
+get_package_version(PACKAGE_VERSION)
+message(STATUS "Opus package version: ${PACKAGE_VERSION}")
+configure_file(config.h.in config.h @ONLY)
+add_definitions(-DHAVE_CONFIG_H)
+include_directories(${CMAKE_CURRENT_BINARY_DIR})
+
+opus_detect_sse(HAVE_SSE)
+opus_detect_neon(HAVE_NEON)
+
+include(CMakeDependentOption)
+cmake_dependent_option(OPUS_PRESUME_SSE
+                       "Use SSE always (requires CPU with SSE)"
+                       ON
+                       "HAVE_SSE;OPUS_PRESUME_SSE"
+                       OFF)
+cmake_dependent_option(OPUS_MAY_HAVE_SSE
+                       "Use SSE if available"
+                       ON
+                       "HAVE_SSE;NOT OPUS_PRESUME_SSE"
+                       OFF)
+
+if(CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64")
+  set(OPUS_PRESUME_NEON ON)
+endif()
+cmake_dependent_option(OPUS_PRESUME_NEON
+                       "Use NEON always (requires CPU with NEON Support)"
+                       ON
+                       "HAVE_NEON;OPUS_PRESUME_NEON"
+                       OFF)
+cmake_dependent_option(OPUS_MAY_HAVE_NEON
+                       "Use NEON if available"
+                       ON
+                       "HAVE_NEON;NOT OPUS_PRESUME_NEON"
+                       OFF)
+
+option(FIXED_POINT "Use fixed-point code (for devices with less powerful FPU)"
+       OFF)
+option(USE_ALLOCA "Use alloca for stack arrays (on non-C99 compilers)" OFF)
+option(BUILD_PROGRAMS "Build programs" OFF)
+
+set_property(GLOBAL PROPERTY USE_FOLDERS ON)
+set_property(GLOBAL PROPERTY C_STANDARD 99)
+
+if(MSVC)
+  add_definitions(-D_CRT_SECURE_NO_WARNINGS)
+  string(REGEX
```

Comment:
Remove it. It is not requirement. CMake script must contain required stuff only.

Reply:
just had some hope that it would encourage to fix the 280 L4 warnings :) so our fork would be easier to merge